pipenv/core.py

```diff
@@ -930,23 +859,17 @@ def do_create_virtualenv(python=None, site_packages=False, pypi_mirror=None):
 
     # Actually create the virtualenv.
     with spinner():
-        try:
-            c = delegator.run(
-                cmd, block=False, timeout=PIPENV_TIMEOUT, env=pip_config,
-            )
-        except OSError:
-            click.echo(
-                "{0}: it looks like {1} is not in your {2}. "
-                "We cannot continue until this is resolved."
-                "".format(
-                    crayons.red("Warning", bold=True),
-                    crayons.red(cmd[0]),
-                    crayons.normal("PATH", bold=True),
-                ),
-                err=True,
-            )
-            sys.exit(1)
-    click.echo(crayons.blue(c.out), err=True)
+        c = delegator.run(
+            cmd, block=False, timeout=PIPENV_TIMEOUT, env=pip_config,
+        )
+    c.block()
```

Comment:
If we don't block here and we pass `block=False` we can't read the output...

Reply:
I think it should be indented inside the `with` block?

Reply:
makes sense to me, I just put it there because of no reason in particular